zingolib/src/lightclient/send.rs

```diff
@@ -23,35 +22,23 @@ impl LightClient {
         ) + 1)
     }
 
+    #[deprecated(note = "superceded by quick_send")]
```

Comment:
my opinion is that do_send should be left intact and deprecated when zip317 is fully tested and a viable alternative. as far as testing goes we dont use do_send we use send_from_send_inputs (a wrapper on do_send), and this function should be replaced by quick_send_from_send_inputs throughout. if do_send is heading towards deprecation it makes sense to make this switch in our testing now anyway